### ISSUE TITLE
[a11y] Cannot use Accession QCBX Search

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1241]
**Describe the bug**
When attempting to use the accession QCBX on the CO form, this error appears:

`TypeError: Cannot read properties of undefined (reading 'getFieldErrors')

    at S (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:52078)
    at z (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:60266)
    at q (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:60694)
    at ComboBox (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:265238)
    at Ue (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:266569)
    at Field (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:266780)
    at He (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:268955)
    at div
    at DataEntry.Cell
    at div
    at div
    at A (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:41575)
    at div
    at C (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:226329)
    at k (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:226024)
    at form
    at Form
    at div
    at div
    at d (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:2352205)
    at E (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:2343269)
    at div
    at n (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:2374789)
    at n (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:2369198)
    at h (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:126416)
    at S (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:220599)
    at div
    at O (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:161654)
    at QueryComboBox (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:265631)
    at Ue (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:266569)
    at Field (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:266780)
    at He (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:268955)
    at div
    at DataEntry.Cell
    at div
    at div
    at A (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:41575)
    at div
    at C (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:226329)
    at k (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:226024)
    at form
    at Form
    at section
    at Container.Generic
    at q (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:211924)
    at Z (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:213275)
    at $ (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:216098)
    at s (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:187453)
    at g (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:77695)
    at b (https://a11y.test.specifysystems.org/static/js/main.c7cf1aa962447f50c12a.bundle.js:2:64203)`

**To Reproduce**
1. Go to the CO form
2. Try to open the Accession form
3.  See error

**Expected behavior**
This should not happen.

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/163587907-938bb17d-dbd9-4c1a-bb18-c0a02e86686a.png)

![image](https://user-images.githubusercontent.com/37256050/163587891-774abe3d-2cfe-4492-b432-ac0e3208d754.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Edge

**Comment by maxxxxxdlp** [comment]
Does not seem to happen anymore
